src/Esfa.Vacancy.Api.Types/BadRequestError.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Esfa.Vacancy.Api.Types
+{
+    /// <summary>
+    /// Error details resulting from a bad request (HTTP STATUS 400)
+    /// </summary>
+    public class BadRequestError
+    {
+        /// <summary>
+        /// Vacancy API error code
+        /// </summary>
+        public string ErrorCode { get; set; }
```

Comment:
Could this be an int?

Reply:
These error codes are populated from ValidationFailure.ErrorCode which is a string. No need to change them to ints before changing back to string in the json serialisation.

The domain error codes are also defined as strings - mainly because ValidationFailure.ErrorCode is a string - I was having to put int.ToString() everywhere which sucked so changed them to strings.

Reply:
👍 So ValidationFailure is not one of our classes, it is a class from FluentValidation?